sensu/config_test.go

```diff
@@ -86,30 +81,53 @@ func validateClient(actualClient *stdClient.Client, t *testing.T) {
 		actualClient.Subscriptions,
 		expectedClient.Subscriptions,
 	) {
-
 		t.Errorf(
-			"Expected client subscriptions to be \"%v\" but got \"%v\" instead!",
+			"Expected client subscriptions to be \"%#v\" but got \"%#v\" instead!",
 			expectedClient.Subscriptions,
 			actualClient.Subscriptions,
 		)
 	}
 }
 
-func expectedClientFromConfig(t *testing.T) {
-	config := Config{config: &configPayload{Client: expectedClient}}
+var dummyClient = &stdClient.Client{
```

Comment:
could you define this variable on top of the file right after the `package` block